account_creation_page.py

```diff
@@ -0,0 +1,60 @@
+import streamlit as st
+import re
+
+
+with st.form("account_creation_form"):
+
+    st.header("Social Stock Analyzer")
+    st.subheader("Create your Account")
+
+    user_name = st.text_input('',placeholder='Username')
+    f_name = st.text_input('',placeholder='Firstname')
```

Comment:
Should be two words.

Reply:
What do you mean, by two words? change f_name to first_name?

Reply:
it should prompt the user "First name" instead of "Firstname"

Reply:
I see. Thank you! I changed.